.github/workflows/test-rc-release.yaml

```diff
@@ -0,0 +1,152 @@
+          gh pr create --base ${{ inputs.base_git_rev }} \
+                       --title "[DO NOT MERGE] Test RC provider packages" \
+                       --body "${{ inputs.issue_url }}" --fill
+          echo "git_rev=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
+    outputs:
+      rc_testing_branch: ${{ steps.create_rc_branch.outputs.rc_testing_branch }}
+
+  export-rc-testing-branch-name:
+    needs: [validate-manual-input, create-branch-for-testing-rc-release]
+    if: |
+      always() &&
+      needs.create-branch-for-testing-rc-release.result == 'success' ||
+      (needs.validate-manual-input.result == 'success' && inputs.rc_testing_branch )
+    runs-on: 'ubuntu-20.04'
+    steps:
+      - name: export rc_testing_branch
+        id: export-rc-testing-branch-name-step
+        run: |
+          if [ "${{ inputs.rc_testing_branch }}" == "" ]; then
+            rc_testing_branch=${{ needs.create-branch-for-testing-rc-release.outputs.rc_testing_branch }}
+          else
+            rc_testing_branch=${{ inputs.rc_testing_branch }}
+          fi
+          echo "rc_testing_branch=$rc_testing_branch" >> $GITHUB_OUTPUT
+    outputs:
+      rc_testing_branch: ${{ steps.export-rc-testing-branch-name-step.outputs.rc_testing_branch }}
+
+  deploy-rc-testing-branch-to-astro-cloud:
+    needs: export-rc-testing-branch-name
+    if: |
+      always() &&
+      needs.export-rc-testing-branch-name.result == 'success'
+    uses: ./.github/workflows/deploy-to-astro-cloud-reuse-wf.yaml
+    with:
+      git_rev: ${{ needs.export-rc-testing-branch-name.outputs.rc_testing_branch }}
+      environment_to_deploy: 'providers-integration-tests'
+    secrets:
+      docker_registry: ${{ secrets.DOCKER_REGISTRY }}
+      organization_id: ${{ secrets.ORGANIZATION_ID }}
+      deployment_id: ${{ secrets.PROVIDER_INTEGRATION_TESTS_DEPLOYMENT_ID }}
+      astronomer_key_id: ${{ secrets.PROVIDER_INTEGRATION_TESTS_ASTRONOMER_KEY_ID }}
+      astronomer_key_secret: ${{ secrets.PROVIDER_INTEGRATION_TESTS_ASTRONOMER_KEY_SECRET }}
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+  wait-for-deployment-to-be-ready:
+    needs: deploy-rc-testing-branch-to-astro-cloud
+    if: |
+      always() &&
+      needs.deploy-rc-testing-branch-to-astro-cloud.result == 'success'
+    runs-on: 'ubuntu-20.04'
+    steps:
+      - name: Sleep for 30 minutes
+        run: |
+          echo "Current timestamp is" `date`
+          echo "Sleeping for 1800 seconds (30 minutes)"
+          echo "allowing the deployed image to be updated across all Airflow components.."
+          sleep 1800
+
+  trigger-master-dag:
+    needs: wait-for-deployment-to-be-ready
+    if: |
+      always() &&
+      needs.wait-for-deployment-to-be-ready.result == 'success'
+    runs-on: 'ubuntu-20.04'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.create-branch-for-testing-rc-release.outputs.rc_testing_branch }}
```

Comment:
Shouldn't we use here `export-rc-testing-branch-name` outputs?

Reply:
Yes, fortunately, this won't break our current running workflow🎉. Let me send a PR to fix it now 💪

Reply:
Thanks for your suggestion! This is addressed on https://github.com/astronomer/astronomer-providers/pull/1135